docs/sdks_03_core.md

```diff
@@ -0,0 +1,109 @@
+---
+id: sdks_core
+title: Safe Core SDK
+sidebar_label: Safe Core SDK
+---
+
+## Introduction
+
+The Safe Core SDK facilitates the interaction with the [Gnosis Safe contracts](https://github.com/gnosis/safe-contracts).
+
+It only supports Safe contracts `v1.2.0` and `ethers.js` `v5` so far.
```

Comment:
also contract priort to 1.2.0, right? (e.g. 1.1.1)

Reply:
I think everything should work with v1.1.1 too, except that that the method `isModuleEnabled` in ModulManager.sol does not exist in v1.1.1